packages/api/src/schema/db-schema.yaml

```diff
@@ -949,6 +968,40 @@ components:
                     - background_mist
                     - fallback_external
                     - external
+            clip:
+              properties:
+                clipStrategy:
+                  type: object
+                  description: >-
+                    Strategy to use for clipping the asset. If not specified,
+                    the default strategy that Catalyst is configured for will be
+                    used. This field only available for admin users, and is only
+                    used for E2E testing.
+                  additionalProperties: false
+                  properties:
+                    startTime:
+                      type: number
+                      description: Start time of the clip in milliseconds
+                    endTime:
+                      type: number
+                      description: End time of the clip in milliseconds
+                    playbackId:
+                      type: string
+                      description: Playback ID of the stream or asset to clip
+                catalystPipelineStrategy:
```

Comment:
Wonder if we can/should just reference some common field to avoid repeating this